src/ucp/proto/proto_select.h

```diff
@@ -31,6 +31,11 @@
 #define UCP_PROTO_SELECT_OP_FLAG_INTERNAL (UCP_PROTO_SELECT_OP_FLAGS_BASE << 1)
 
 
+/* Select eager/rendezvous protocol for Active Message sends */
+#define UCP_PROTO_SELECT_OP_FLAG_AM_EAGER (UCP_PROTO_SELECT_OP_FLAGS_BASE << 2)
+#define UCP_PROTO_SELECT_OP_FLAG_AM_RNDV  (UCP_PROTO_SELECT_OP_FLAGS_BASE << 3)
```

Comment:
can this operation flag be re-used for TAG protocol then?
if yes - wdyt to drop `AM` suffix from their names?

Reply:
guess these flags can be used in such way.
however, we can force eager/rndv selection for AM in current protocol implementation, but not fot TAG

Reply:
> however, we can force eager/rndv selection for AM in current protocol implementation, but not fot TAG

I see, thanks